assembl/locale/assembl-v2.pot

```diff
@@ -3871,6 +4046,13 @@ msgstr ""
 msgid "debate.voteSession.postSuccess"
 msgstr ""
 
+#: messages.json
+#. [debate.voteSession.remainingToken]
+#. defaultMessage is:
+#. %{count} token remaining
```

Comment:
How did this one make the PR? Alas, this one requires using pluralization.

```
...
remainingToken: '%{count} tokens remaining',
remainingToken_0: 'No more tokens left',
remainingToken_1: 'One token remaining'
...
```

Taken from the documentation:
https://www.npmjs.com/package/react-i18nify